app/components/ChangeLog.tsx

```diff
@@ -9,6 +9,9 @@ import {
   ChangeLogEntryEnriched,
 } from "@/types/changeLog";
 
+// used to track the accordion expanded state of the change log
+const localStorageKey = "expandedItem";
```

Comment:
would you mind making this a little more descriptive and specific instead of `"expandedItem"`, like `"crashHistoryCardExpandedItem"`?

Reply:
btw, i created https://github.com/cityofaustin/atd-data-tech/issues/20575 to add in support for version local storage across the app 👍